src/sidebar/components/button.js

```diff
@@ -3,12 +3,27 @@ import propTypes from 'prop-types';
 
 import SvgIcon from '../../shared/components/svg-icon';
 
+/**
+ * @typedef ButtonProps
+ * @prop {string} [buttonText] - The presence of this property indicates that the button will have a visibly-rendered label (with this prop's value). For brevity, when providing `buttonText`, the `title` prop is optional—if `title` is not present, the value of `buttonText` will be used for the button's `title` attribute, as they are typically identical. When this prop is missing, an icon-only button will be rendered.
```

Comment:
Can you reformat long prop descriptions (> 100 chars) like so:

```js
/**
 * @typedef ButtonProps
 * @prop {string} [buttonText] -
 *   The presence of this property indicates that the button will have a
 *   visibly-rendered label (with this prop's value). For brevity, when providing `buttonText`, the
 *   `title` prop is optional—if `title` is not present, the value of `buttonText` will be used for
 *   the button's `title` attribute, as they are typically identical. When this prop is missing, an
 *   icon-only button will be rendered.
 * ...
 */
```

Note the new line after `[buttonText] -` and the two-space indent on subsequent lines of the text.